.codecov.yml

```diff
@@ -0,0 +1,8 @@
+comment:
+  layout: "reach, diff, flags, files"
+  behavior: default
+  require_changes: false # if true: only post the comment if coverage changes
+  require_base: no # [yes :: must have a base report to post]
+  require_head: yes # [yes :: must have a head report to post]
+  branches: # branch names that can post comment
+    - "master"
```

Comment:
just out of curiosity, what dictates whether these `.yml` files go in the `.circleci/` dir or outside?

Reply:
it's contingent on the service we're using. `.circleci` required it to be in a folder, where as `.codecov.yml` by default sets it here. Technically we can override the default location and put everything together, but I figured it's not necessarily for now